pytest_splunk_addon/standard_lib/data_models/Alerts.json

```diff
@@ -17,19 +17,44 @@
             },
             {
                 "name": "body",
+                "type": "optional",
+                "comment":"The body of a message."
+            },
+            {
+                "name": "description",
                 "type": "required",
-                "comment":"The body of a message." 
+                "comment": "The description of the alert event."
             },
             {
                 "name": "dest",
                 "type": "required",
-                "comment":"The destination of the alert message, such as an email address or SNMP trap. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_name."                
+                "comment":"The destination of the alert message, such as an email address or SNMP trap. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_name."
+            },
+            {
+                "name": "dest_type",
+                "type": "optional",
+                "comment": "The type of the destination object, such as instance, storage, firewall."
             },
```

Comment:
Can you please double check if this is provided by A&I framework (similar to fields in https://github.com/splunk/pytest-splunk-addon/blob/3ee0404aa07cd56ee67063c4c86aaeaeb3dbb7be/pytest_splunk_addon/standard_lib/cim_tests/CommonFields.json)?

Reply:
dest_type is not provided by A&I framework, so all fields in https://github.com/splunk/pytest-splunk-addon/blob/3ee0404aa07cd56ee67063c4c86aaeaeb3dbb7be/pytest_splunk_addon/standard_lib/cim_tests/CommonFields.json have the same in description if they are provided by A&I or not.